src/customerCalls/CallForm.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { PureComponent } from 'react'
+import { CallFormButton, CallFormInput, CallFormDataList, CallFormWrapper, CallFormOption } from './components';
+import web3 from "../utils/createAndUnlockWeb3";
+import usingOracleAbi from "../abi/usingOracle.abi";
+
+type State = { query: string };
+type InputEvent = React.ChangeEvent<HTMLInputElement>;
+type SelectEvent = React.ChangeEvent<HTMLSelectElement>;
+
+export const usingOracleContract = new web3.eth.Contract(
+    usingOracleAbi,
+    process.env.REACT_APP_USING_ORACLE_ADDRESS
+);
+
+export default class CallForm extends PureComponent<State> {
+    static defaultProps = { query: '' };
+    state: State = {
+        query: '',
+    }
+
+    handleChange = (event: InputEvent | SelectEvent): void => {
+        event.preventDefault();
+        this.setState({
+            query: event.target.value,
+        })
+    }
+
+    handleSubmit = () => {
+        if (this.state.query === '') {
+            alert('put valid url for call')
+            return;
+        }
+        usingOracleContract.methods.request(this.state.query)
+            .send();
+        this.setState({
+            query: '',
+        })
+    }
+
+    render() {
+        return (
+            <CallFormWrapper>
+                <CallFormInput
+                    value={this.state.query}
+                    onChange={this.handleChange}
+                    list="endpoints"
+                ></CallFormInput>
+                <CallFormDataList id="endpoints">
+                    <CallFormOption
+                        value={'json(https://api.coindesk.com/v1/bpi/currentprice.json).bpi.USD.rate_float'}
```

Comment:
All urls in this file should be taken from conts, ideally from different file.

Reply:
all urls moved to the config.ts file